content/docs/1.0.0/iofogctl/iofogctl.md

```diff
@@ -0,0 +1,209 @@
+
+
+Usage:
+  iofogctl [flags]
+  iofogctl [command]
+
+Available Commands:
+  connect     Connect to an existing ioFog cluster
+  create      Create a resource
+  delete      Delete an existing ioFog resource
+  deploy      Deploy ioFog platform or components on existing infrastructure
+  describe    Get detailed information of existing resources
+  disconnect  Disconnect from an ioFog cluster
+  get         Get information of existing resources
+  help        Help about any command
+  legacy      Execute commands using legacy CLI
+  logs        Get log contents of deployed resource
+  version     Get CLI application version
+
+Flags:
+      --config string      CLI configuration file (default is ~/.iofog.yaml)
+  -h, --help               help for iofogctl
+  -n, --namespace string   Namespace to execute respective command within (default "default")
+
+Use "iofogctl [command] --help" for more information about a command.
+
+```
+
+## Work With Namespaces
+
+All actions performed with iofogctl are scoped to a single namespace. The default namespace ('default') is used if the user does not specify a namespace explicitly in the command. Note that namespaces in `iofogctl` do no have to correspond with namespaces in Kubernetes cluster
+
+Try creating, listing, and deleting namespaces now with the following commands:
+
+```bash
+iofogctl create namespace mynamespace
+iofogctl get namespaces
+iofogctl delete namespace mynamespace
+```
+
+Next we will use the default namespace to create new ioFog resources in.
+
+## Deploy ioFog Stack
+
+If we don't have an existing ioFog stack to connect to, we can use iofogctl to install and provision a new one.
+
+IoFog Controller is the central part of each ioFog stack. `iofogctl` connects to a Controller in order to manage ioFog stack.
+
+We can deploy the ioFog stack to a pre-existing Kubernetes cluster with the following command. Make sure to specify the location of our [kubeconfig file](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) correctly.
+
+```bash
+iofogctl deploy controller my-ctrl --kube-config ~/.kube/conf
+```
+
+Under the hood, `iofogctl` will also deploy ioFog Connector and Kubernetes services, which are another essential part of the ioFog stack
+
+We can now check that the Controller was deployed successfully.
+
+```console
+$ iofogctl get all
+
+CONTROLLER	STATUS		AGE		UPTIME
+my-ctrl		online		-		1s
+
+AGENT		STATUS		AGE		UPTIME
+
+MICROSERVICE	STATUS		AGE
+
+```
+
+It is also possible to query for all Kubernetes resources installed. The namespace `iofogctl` uses is always `iofog`, so we can run `kubectl get all -n iofog` to see all the resources installed on our cluster.
+
+## Connect Existing ioFog Stack With iofogctl
+
+Instead of deploying our own iofogctl stack, we can just connect to another Controller, for example one deployed by Helm. See [helm tutorial](../kubernetes/how-to-helm.html) for in depth dive into Helm.
```

Comment:
Note that you must specify an empty or non-existent namespace when you use the connect command. This is because each cluster should be in its own namespace.